docs/_docs/themes.md

```diff
@@ -4,28 +4,41 @@ title: Themes
+## Understanding gem-based themes
 
-1. To install a theme, first, add the theme to your site's `Gemfile`:
+When you [create a new Jekyll site](/docs/quickstart) (by running the `jekyll new <PATH>` command), Jekyll installs a site that uses a gem-based theme called [Minima](https://github.com/jekyll/minima).
 
-        gem 'my-awesome-jekyll-theme'
+With gem-based themes, some of the site's directories (such as the `assets`, `_layouts`, `_includes`, and `_sass` directories) are stored in the theme-gem, hidden from your immediate view. Yet all of the necessary directories will be read and processed during Jekyll's build process.
 
-2. Save the changes to your `Gemfile`
-3. Run the command `bundle install` to install the theme
-4. Finally, activate the theme by adding the following to your site's `_config.yml`:
+In the case of Minima, you see only the following files in your Jekyll site directory:
 
-        theme: my-awesome-jekyll-theme
+```
+├── Gemfile
+├── Gemfile.lock
+├── _config.yml
+├── _posts
+│   └── 2016-12-04-welcome-to-jekyll.markdown
+├── about.md
+└── index.md
+```
 
-You can have multiple themes listed in your site's Gemfile, but only one theme can be selected in your site's `_config.yml`.
-{: .note .info }
+The `Gemfile` and `Gemfile.lock` files are used by Bundler to keep track of the required gems and gem versions you need to build your Jekyll site.
+
+Gem-based themes make it easy for theme developers to make updates available to anyone who has the theme gem. When there's an update, theme developers push the update to RubyGems.
+
+If you have the theme gem, you can (if you desire) run `bundle update` to update all gems in your project. Or you can run `bundle update <theme>`, replacing `<theme>` with the theme name, such as `minima`, to just update the theme gem. Any new files or updates the theme developer has made (such as to stylesheets or includes) will be pulled into your project automatically.
+
+The goal of gem-based themes is to allow you to get all the benefits of a robust, continually updated theme without having all the theme's files getting in your way and over-complicating what might be your primary focus: creating content.
 
 ## Overriding theme defaults
 
-Jekyll themes set default layouts, includes, and stylesheets, that can be overridden by your site's content. For example, if your selected theme has a `page` layout, you can override the theme's layout by creating your own `page` layout in the `_layouts` folder (e.g., `_layouts/page.html`).
+Jekyll themes set default layouts, includes, and stylesheets. However, you can override any of the theme defaults with your own site content.
 
-Jekyll will look first to your site's content, before looking to the theme's defaults, for any requested file in the following folders:
+For example, if your selected theme has a `page` layout, you can override the theme's layout by creating your own `page` layout in the `_layouts` directory (for example, `_layouts/page.html`).
```

Comment:
> (for example, `_layouts/page.html`)

This is a tricky case of "e.g." vs "i.e." and I think it's "i.e." here, because you're elaborating and giving specific information about a pre-existing example. This would read better to me:

> by creating your own `page` layout in the `_layouts` directory, i.e. `_layouts/page.html`.

Feel free to replace "i.e." with something less Latin if you would prefer.